### DISCUSSION TITLE
how can i propose and add consume feature for android for this laravel-in-app-purchases

### DISCUSSION THREAD

**Discussion by lolitototo** [imdhemy/laravel-in-app-purchases#361] [Q&A] [answered]
Hi,

It's the first time i want to add a feature to a package on github (i'm noob in github and my english is not good), i have read some pages and websites on how to do this. The problem is that when i duplicate the package to put my new feature and to propose my feature to your repo, the repo is not the same that my laravel folder, there are not the google-play-billing and appstore-iap folders, only the laravel-purchases.
The feature need some lines in laravel-purchases and some lines in google-play-billing folder. (feature = add consume item on android).
i didnt see that feature at the moment in this package but maybe i didnt find it.

TY

**Comment by imdhemy** [comment, marked as answer]
Hello @lolitototo 
This package uses two packages under the hood. You need to fork both of them and add you changes separately. This will end up into submitting 3 merge requests.

https://github.com/imdhemy/google-play-billing

https://github.com/imdhemy/appstore-iap

**Reply by lolitototo** [reply]
Ty, i did the pull request.
Thx for your packages.
